client/lib/post-normalizer/rule-wait-for-images-to-load.js

```diff
@@ -43,7 +43,12 @@ function promiseForImage( image ) {
 	} );
 }
 
-const promiseForURL = flow( imageForURL, promiseForImage );
+const promiseForURL = function( urlOrPromise ) {
+	if ( urlOrPromise instanceof Promise ) {
```

Comment:
yup, when we already know the size. I was trying to do the minimal thing to the image loading pipeline here, so I represented a "known" image as an already resolved promise, resolving it with a duck-typed image object. Not super clean, but it slips through the flow nicely.